typescript-light-bulb/main.ts

```diff
@@ -0,0 +1,18 @@
+let on: boolean = false;
+
+const $container = document.querySelector('.container');
+if (!$container) throw new Error('.container query failed!');
+const $lightbulb = document.querySelector('.lightbulb');
+if (!$lightbulb) throw new Error('.lightbulb query failed!');
+
+$container.addEventListener('click', () => {
+  if (on === false) {
+    on = true;
+    $lightbulb.className = 'lightbulb light-on';
```

Comment:
This could be done with a more compact condition expression. If you want to check if `on` is false, we can check if `on` is _not_ truthy.

`if (!on)` will run the same condition as `if (on === false)`